_scss/_utilities.scss

```diff
@@ -189,6 +189,8 @@ input[type=text] {
 }
 
 @media print {
+    #announcement-bar,
```

Comment:
The announcement bar is added by our marketing team through Google Tag Manager; I don't think we should add this here, because we don't know in advance what ID they might be using (now, and in future); but I can ask the marketing team if they could add a class or something else to banners to prevent them being printed.

```suggestion
```

Reply:
Yes, that would be great. (Please pardon my late reply.)

I think it'd be very reasonable to have it hidden when converting the documentation pages to PDF for printing. Having such announcement on every other printed page currently is, indeed, wasting paper and reading time.